File: javascript/1020.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var inputV = lines.shift();
var years = Math.floor(inputV / 365);
var months = Math.floor((inputV - years * 365) / 30);
var days = inputV - (years * 365 + months * 30);
console.log(years + " ano(s)");
console.log(months + " mes(es)");
console.log(days + " dia(s)");

File: C/1020.c
#include <stdio.h>
#include <math.h>

int main() {
    int input,y,m,d;
    scanf("%d", &input);
    y=floor(input/365.0);
    m=floor((input - y * 365.0)/30.0);
    d=input - (y * 365.0 + m * 30.0);
    printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n",y,m,d);
    return 0;
}